types/vanillajs-datepicker/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for vanillajs-datepicker 1.1
```

Comment:
```suggestion
// Type definitions for vanillajs-datepicker 1.2
```
is this 1.1 specific or should be aligned with latest published (1.2)?